tests/jballerina-unit-test/src/test/resources/test-src/query/query_action_or_expr.bal

```diff
@@ -915,6 +915,28 @@ function testQueryActionWithQueryExpression() {
     assertEquality([4, 25], res2);
 }
 
+
+function testQueryActionWithRegexpLangLibs() {
+    string[] res = [];
+
+    from var item in ["a", "aab", "bc", "ac"] 
+    do {
+        if re `a.*`.isFullMatch(item) {
+            res.push(item);
+        }
+    };
+    
+    assertEquality(["a", "aab", "ac"], res);
+}
+
+function testQueryExprWithRegExpLangLibs() {
+    string[] res = from var item in ["a", "aab", "bc", "ac"] 
+    where re `a.*`.isFullMatch(item)
+    select item;
+    
+    assertEquality(["a", "aab", "ac"], res);
+}
```

Comment:
Add a test where a interpolated regex is being used within the query

Reply:
Added.